app/uk/gov/hmrc/bindingtariffclassification/search/Search.scala

```diff
@@ -40,13 +40,15 @@ case class Filter
   assigneeId: Option[String] = None,
   statuses: Option[Set[CaseStatus]] = None,
   traderName: Option[String] = None,
-  minDecisionEnd: Option[Instant] = None
+  minDecisionEnd: Option[Instant] = None,
+  commodityCode: Option[String] = None,
+  goodDescription: Option[String] = None
 )
 
 case class Sort
 (
   field: SortField,
-  direction: SortDirection = SortDirection.DESCENDING
```

Comment:
Leave this as it was please. If you specify a sort field then we need a default direction, others we can’t sort. The whole sort block is optional

Reply:
Also the defaults should be inside the Scala models, otherwise it’s a pain to find them

Reply:
I will change the default sorting to ASCENDING, as it is the most natural sorting (think for example about alphabetic order, ids, etc)